includes/class-wcpdf-settings-callbacks.php

```diff
@@ -80,6 +80,10 @@ public function text_input( $args ) {
 			$type = 'text';
 		}
 
+		if (empty($current) && isset($default)) {
+			$current = $default;
+		}
```

Comment:
is this not already handled here?
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/c64fa0e23eacdd55dfb8df1b14fd8a1b7b27766e/includes/class-wcpdf-settings-callbacks.php#L535-L538

Reply:
6f9fe5d137ac5544a0ef87a15a7e86681e4785f0

Reply:
I think it makes more sense to do this in the `normalize_settings_args` method so that all callbacks can use it?

Reply:
Yes, i will fix.